src/rgw/rgw_cr_rados.cc

```diff
@@ -690,12 +692,55 @@ int RGWAsyncFetchRemoteObj::_send_request(const DoutPrefixProvider *dpp)
     if (counters) {
       counters->inc(sync_counters::l_fetch_err, 1);
     }
-  } else if (counters) {
-    if (bytes_transferred) {
-      counters->inc(sync_counters::l_fetch, *bytes_transferred);
-    } else {
-      counters->inc(sync_counters::l_fetch_not_modified);
-    }
+  } else {
+      // r >= 0
+      if (bytes_transferred) {
+		ldpp_dout(dpp, 0) << "sending sync notification" << dendl;
```

Comment:
can we drop this log message? in most cases, notifications for sync won't be enabled on the bucket, so this message would be misleading

Reply:
:+1:
It was here for debugging right now but it can be removed once the commit is finalized for the PR

Reply:
sounds good